src/store/modules/phyx.js

```diff
@@ -108,4 +113,165 @@ export default {
       if (has(payload, 'orcid')) Vue.set(state.currentPhyx, 'curatorORCID', payload.orcid);
     },
   },
+  actions: {
+    createPhylogenyFromOpenTree({ commit, state }) {
+      // Create a new, empty phylogeny from the Open Tree of Life using the /induced_subtree endpoint
+      // at https://github.com/OpenTreeOfLife/germinator/wiki/Synthetic-tree-API-v3#induced_subtree
+
+      function createOTTPhylogenyWithCitation(phylogeny) {
+        // Create an OTT phylogeny after querying for the synthetic tree ID.
+        jQuery.ajax({
+          type: 'POST',
+          url: 'https://api.opentreeoflife.org/v3/tree_of_life/about',
```

Comment:
Shouldn't we make the API URL path prefix a constant, so it's not repeated in multiple places?

Reply:
Fixed in f8fa3f8b70e2b36973850a6cdc7daa977d4f53a7.